tests/sample/test_sampler.py

```diff
@@ -0,0 +1,96 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
do not create a new sample folder, this kind of test won't be ran by defaut. Please move to singcard folder